### DISCUSSION TITLE
Retrieving Errors from the response

### DISCUSSION THREAD

**Discussion by jhurder-cvet** [IlyaMatsuev/Apex-GraphQL-Client#13] [Q&A] [answered]
Hello - just getting started with GraphQL-Client today. I am able to send a request, however it has an error in the request as the `hasErrors` property is true after I send the request. Is there a way to retrieve these errors?

**Comment by jhurder-cvet** [comment, marked as answer]
OOps - sorry! I didn't notice the getErrors() method

**Reply by IlyaMatsuev** [reply]
Hi, no problem!
You can also check the docs (https://github.com/IlyaMatsuev/Apex-GraphQL-Client/tree/main/docs/types#graphqlresponse) to see all available methods

**Reply by jhurder-cvet** [reply]
Fantastic - thanks again!

**Comment by IlyaMatsuev** [comment]
Hello!
Thanks for using it 😁
No problem! You can also check the docs (
https://github.com/IlyaMatsuev/Apex-GraphQL-Client/tree/main/docs/types#graphqlresponse)
to see all available methods

Best Regards,
Ilya Matsuev

вт, 13 сент. 2022 г. в 14:47, jhurder-CVET ***@***.***>:
